resources/js/features/games/components/TopAchieversMainRoot/TopAchieversList/TopAchieversList.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { AwardType } from '@/common/utils/generatedAppConstants';
+import { render, screen } from '@/test';
+import { createFactory } from '@/test/createFactory';
+import { createGame, createPaginatedData, createPlayerBadge, createUser } from '@/test/factories';
+
+import { TopAchieversList, TopAchieversListContainerTestId } from './TopAchieversList';
+
+export const createAchiever = createFactory<App.Platform.Data.GameTopAchiever>((faker) => {
+  return {
+    rank: 1,
+    user: createUser(),
+    score: faker.number.int({ min: 1, max: 100 }),
+    badge: null,
+  };
+});
```

Comment:
I think this should probably live in a _test/factories/createGameTopAchiever.ts_ file. When the game page is eventually converted to React, I suspect it'll use the same `GameTopAchieverData` DTO for the component that leads to this page.

In other words, you should be able to also import `createGameTopAchiever` from `@/test/factories`. You'll need to add it to _test/factories/index.ts_ and it should just work.

It's ok for the new _createGameTopAchiever.ts_ file to import `createUser`.

Reply:
done.